### PR TITLE
Nba app radix

### PR DESCRIPTION
**Pull request by tgberkeley**

### CODE REVIEW

File: .github/workflows/app_harness.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: app-harness
 env:
-  REFLEX_VERSION: "==0.3.8"
+  REFLEX_VERSION: "==0.3.9"
```
Comment (martinxu9) [inline .github/workflows/app_harness.yml:3]: I'll make another PR for it, we can keep only the app changes in this PR

File: nba_radix/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+reflex>=0.3.8
```
Comment (martinxu9) [inline nba_radix/requirements.txt:1]: 0.4.0?

**Review by martinxu9 — COMMENTED** [review]
also let's keep the name as nba instead of nba_radix

File: .github/workflows/app_harness.yml (hunk 1 above)
Comment (tgberkeley) [reply .github/workflows/app_harness.yml:3]: What about the old nba app?

**Comment by tgberkeley** [comment]
This PR should be all good to go now

**Review by martinxu9 — APPROVED** [review]

**Pull request merged**